Dockerfile.dev

```diff
@@ -9,14 +9,11 @@ WORKDIR /go/src/github.com/netlify/gotrue
 
 # Pulling dependencies
 COPY ./Makefile ./go.* ./
-
-# Production dependencies
 RUN make deps
 
-# Development dependences
-RUN go get -u github.com/Thatooine/go-test-html-report
```

Comment:
At one point I also had this running tests and opening a test report, hence this dependency existing.